storybook/src/docs/borders.docs.mdx

```diff
@@ -2,9 +2,9 @@
 
 import { Meta } from "@storybook/blocks";
 
-<Meta title="Docs/Design Guidelines/Borders" />
+<Meta title="Foundation/Design Tokens/Border" />
```

Comment:
'Foundation' is a bit jargon-y imo. How about 'Building blocks'? Or even simpler: 'Base'?

Reply:
Jargon is not necessarily bad – it may indicate that it’s a term the industry agreed on, and that is’s being used broadly so people outside of the community will recognise it as well.

The term ‘Building blocks’ fits the component level better – those are the things you build with. ‘Base’ is very generic.

We can consider the direction of ‘Identity’ or ‘Branding’, but I think Foundation is fine as well.

Reply:
> that is’s being used broadly so people outside of the community will recognise it as well.

I think that is kind of the opposite of [the definition of jargon](https://www.oxfordlearnersdictionaries.com/definition/american_english/jargon) ;)

That's my point here: I don't think 'Foundation' is a term easily understood by people outside of design systems-fans.

Reply:
Argh I should have just written that second and third paragraph :)

Foundations are known as that helpful layer below buildings, no? 😝